pkg/sql/logictest/testdata/logic_test/pg_catalog

```diff
@@ -1221,10 +1222,20 @@ SELECT
   ORDER BY 3, 4, 5
 ----
 depend_catalog  source_catalog  source        depend                 column_name  deptype
-pg_class        pg_class        depend_view   view_dependingon_view  b            n
```

Comment:
@rafiss I was thinking if I should keep this record at pg_depend as we have already implemented and this might break someone's query.